sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S2175.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
The RSPEC PR was very old and some of the metadata was not compatible with today's format anymore. I rebased the PR on master, it should be good now. However, we need to update the rule metadata again.